src/sage/features/__init__.py

```diff
@@ -264,6 +293,109 @@ def resolution(self):
         self._cache_resolution = "\n".join(lines)
         return self._cache_resolution
 
+    def joined_features(self):
+        r"""
+        Return a list of features joined with ``self``.
```

Comment:
perhaps "features that ``self`` is the join of"

Reply:
Done!